src/components/editor/InputLiteral.jsx

```diff
@@ -169,7 +146,9 @@ export class InputLiteral extends Component {
 
   makeAddedList = () => {
     let formInfo = this.props.formData
-    if (formInfo == undefined) return
+    // console.log(`makeAddedList ${this.props.propertyTemplate.propertyLabel}`)
+    // console.warn(formInfo)
```

Comment:
do we want to keep these console.log statements?

Reply:
Removed in latest commit.